src/plugins/open/open.c

```diff
@@ -91,7 +91,7 @@ extern int verbose;
 static int
 ipmi_openipmi_open(struct ipmi_intf * intf)
 {
-	int i = 0;
+	int i = 1;
```

Comment:
Good catch. This ioctl is listed in manpages as 'Stub only', but apparently it not a stub anymore.
Nonetheless, I hate this `i` name for the variable. That is a nice name for an iterator, but is awful for an ioctl parameter. Since you're touching this line, it's a good moment to rename the variable to something meaningful like `receive_events`. Also, as `ipmitool` is now built according to C11, you can safely move the definition closer to the `ioctl()` using it.

I would also prefer `TRUE` used instead of `1`. That macro is available via `ipmitool/helper.h`.